pontoon/base/views.py

```diff
@@ -741,6 +741,21 @@ def update_translation(request):
         })
 
 
+@utils.require_AJAX
+def update_tour_status(request):
+    """Perform quality checks and return a list of any failed ones."""
+    try:
+        tour_status = request.POST['tour_status']
```

Comment:
You should validate that this is an integer.

Reply:
I think a django form is the way to go then ;-)